### DISCUSSION TITLE
Dependency dashboard is recreated each time Renovate runs if the user overwrites the bot's token using hostRules

### DISCUSSION THREAD

**Discussion by meriouma** [renovatebot/renovate#14980] [Request Help]
We're using the Renovate GitHub hosted app. We had to setup a host rule to allow composer to fetch our private packages :
```
   {
      "hostType": "github",
      "matchHost": "https://api.github.com/",
      "encrypted": {
        "token": "..."
      }
    }
```

Since then, the "active" user is now the user associated with the PAT instead of `renovate[bot]`, and it seems the `findIssue(Dependency Dashboard)` is unable to find the issue, and recreates the issue every time. I'm not sure what we did wrong?

**Comment by viceice** [comment]
use hostType=composer

**Reply by meriouma** [reply]
Aah, that makes sense! Thank you!
